code/model/SiteTreeFileExtension.php

```diff
@@ -109,7 +109,7 @@ public function BackLinkTrackingCount() {
 	 */
 	public function onAfterDelete() {
 		// Skip live stage
-		if(\Versioned::current_stage() === \Versioned::get_live_stage()) {
+		if(\Versioned::get_stage() === Versioned::LIVE_STAGE) {
```

Comment:
Turns out it doesn't matter, but yes it SHOULD be consistent.

I changed IDEs since startinging this project haha. :D